sw/device/tests/sim_dv/ast_clk_rst_inputs.c

```diff
@@ -204,15 +205,22 @@ void init_units() {
  *  configure adc module
  */
 static void configure_adc_ctrl(const dif_adc_ctrl_t *adc_ctrl) {
+  int WakeUpTimeAonCycles;
+  int PowerUpTimeAonCycles;
```

Comment:
```suggestion
  int wake_up_time_aon_cycles;
  int power_up_time_aon_cycles;
```
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#naming

Reply:
Fixed.

Reply:
Fixed.

Reply:
@matutem I see that the kokoro fails, but I don't see why.
can you  please help me to understand what is the reason for this fail.

Reply:
that was an intermittent error - I reran kokoro and it is passing now.